ditto/cli.py

```diff
@@ -98,32 +109,28 @@ def metric(ctx, **kwargs):
             "Cannot read from format '{}'".format(kwargs["from"])
         )
 
-    if kwargs["input"] is None:
```

Comment:
Click will check this automatically, so I removed the unnecessary code in a few places.